components/admin/requests/SelectedPermitHolderCard.tsx

```diff
@@ -11,18 +11,46 @@ import {
   Flex,
   Wrap,
   Badge,
+  Spinner,
+  Center,
 } from '@chakra-ui/react'; // Chakra UI
 import { ApplicantData } from '@tools/pages/admin/permit-holders/types'; // Applicant data type
 import { formatDate } from '@lib/utils/format'; // Date formatter util
 
 type SelectedPermitHolderCardProps = {
   readonly applicant: ApplicantData;
+  readonly loading: boolean;
 };
 
 export default function SelectedPermitHolderCard(props: SelectedPermitHolderCardProps) {
-  const { applicant } = props;
+  const { applicant, loading } = props;
   const { hasCopied, onCopy } = useClipboard(applicant?.email || '');
 
+  if (loading) {
+    return (
+      <GridItem
```

Comment:
Since this doesn't span the grid layout, we should be able to just use `VStack` here and remove `display="flex"` and `flexDirection="column"`